src/client/components/CostWidget.js

```diff
@@ -16,4 +17,9 @@ function CostWidget({ currentCost, remainingCost }) {
   );
 }
 
+CostWidget.propTypes = {
+  currentCost: PropTypes.string,
+  remainingCost: PropTypes.string
```

Comment:
I dont think currentCost and remainingCost is a string. It should be number. Please check with Zaki.

Reply:
Correct. Any cost fields should be a number.

Reply:
We will probably add the ability to display currency, but that should be a separate prop instead of making the number into a string.